src/utils/searchCache.ts

```diff
@@ -31,7 +31,16 @@ const saveCache = (newCache: TCacheSearch) => {
   window.localStorage.setItem(CACHE_NAME, JSON.stringify(newCache));
 };
 
+const clearOldCache = () => {
+  const cache = getCache();
+  const newCache = {
+    cache: cache.cache.filter((search) => search.timestamp > Date.now() - 1000 * 60 * 60 * 24),
```

Comment:
Reads a little "magical" - can we have a descriptive const for `1000 * 60 * 60 * 24` ?